release-notes/2.2/releases.json

```diff
@@ -4,7 +4,7 @@
   "latest-release-date": "2019-11-19",
   "latest-runtime": "2.2.8",
   "latest-sdk": "2.2.207",
-  "support-phase": "maintenance",
+  "support-phase": "eol",
```

Comment:
do you need to make the same change in https://github.com/dotnet/core/blob/master/release-notes/releases-index.json#L35 as I had in my PR?